introduction.md

```diff
@@ -0,0 +1 @@
+Hi my name is Perry.  This is the second attempt to make this work.  I feel that the instructions are not concise enough for a beginner such as myself.  Either that or I am not cut out for this line of work.  My skillsets are with Java, Javascript, and a little front end HTML and CSS.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.